core/components/atoms/dropdown/DropdownList.tsx

```diff
@@ -679,6 +700,8 @@ const DropdownList = (props: OptionsProps) => {
     }
   };
 
+  const shouldSearchRender = (withSearch || props.async) && (!_isNoResultReturned || errorType === 'NO_RECORDS_FOUND');
```

Comment:
Please change this variable name

Reply:
variable name changed to **enableSearch**.